lib/portlayer/exec/container.go

```diff
@@ -175,7 +175,9 @@ func (c *Container) Commit(ctx context.Context, sess *session.Session, h *Handle
 		c.vm = vm.NewVirtualMachine(ctx, sess, res.Result.(types.ManagedObjectReference))
 		c.State = StateCreated
 		commitEvent = events.ContainerCreated
-
+		// align the handle state w/the container
+		h.State = &c.State
```

Comment:
Actually is wondering why not just align the status here. Looks like h.container.Stop is not cheap.